packages/styles/src/navigation.scss

```diff
@@ -101,9 +101,29 @@ $navListContainerSubmenu: #{$block}__list-container--submenu;
     }
   }
 
+  @mixin _fd-vertical-nav() {
+    &--vertical {
+      @content;
+    }
+  }
+
+  @mixin _fd-horizontal-nav() {
+    &--horizontal {
+      @content;
+    }
+  }
```

Comment:
I am against these mixins. The code refactoring later is not very fun. Please use SCSS variables